test/base/test_returns.py

```diff
@@ -70,7 +71,7 @@ def test_episodic_returns(size=2560) -> None:
         truncated=np.array([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]),
         rew=np.array([101, 102, 103.0, 200, 104, 105, 106, 201, 107, 108, 109, 202]),
     )
-    for b in batch:
+    for b in iter(batch):
```

Comment:
With the newest PR merged, `__iter__` was added to Batch. It should be ok to remove the explicit iter now, right?

Reply:
Correct.